gramps/gen/lib/serialize.py

```diff
@@ -66,7 +66,65 @@ def __str__(self):
             self["_object"] = from_dict(self)
         return str(self["_object"])
 
+    def __int__(self):
+        if "_object" not in self:
+            self["_object"] = from_dict(self)
```

Comment:
```suggestion
    def __get_object(self):
        if "_object" not in self:
            self["_object"] = from_dict(self)

    def __int__(self):
        self.__get_object()
```

Given the number of places you repeats these two lines, consider extracting into a private method